### DISCUSSION TITLE
How to style with tailwind?

### DISCUSSION THREAD

**Discussion by Blankeos** [faceless-ui/modal#48] [General]
Great library! But I'm not using SCSS in my project but I just wanted to try this out instead of headless-ui. How can I style this with tailwind?

**Comment by jacobsfletch** [comment]
Hi! You can absolutely use tailwind by simply adding the tailwind classes to each of the elements you wish to style. All Faceless UI components fully expose their DOM, so you have complete control over the markup.

**Comment by jacobsfletch** [comment]
@Blankeos does this answer your question? I'm going to go ahead and convert this issue to a discussion so we can keep the conversation going!
